source/common/stats/thread_local_store.cc

```diff
@@ -24,11 +25,13 @@ namespace Stats {
 const char ThreadLocalStoreImpl::MainDispatcherCleanupSync[] = "main-dispatcher-cleanup";
 
 ThreadLocalStoreImpl::ThreadLocalStoreImpl(Allocator& alloc)
-    : alloc_(alloc), default_scope_(createScope("")),
+    : alloc_(alloc), default_scope_(ThreadLocalStoreImpl::createScope("")),
```

Comment:
why is it necessary to qualify createScope() now, out of curiosity?

Reply:
This was to fix clang_tidy. createScope() is virtual, and it didn't like that we were calling a virtual method in the constructor, probably because if there was a sub-class of ThreadLocalStoreImpl that had an override for createScope(), the wrong one would be called here.